### DISCUSSION TITLE
I don't know how to get data from Mongodb atlas.

### DISCUSSION THREAD

**Discussion by RyanBreathings** [apostrophecms/apostrophe#3640] [Q&A]
Hello

I'm making a bulletin board page.
I want to take the data stored in Mongodb atlas and output it on the screen.
Currently, Mongodb atlas is connected, but I don't know how to get data on the page I created.
Can you tell me how to do it?

**Comment by abea** [comment]
Is this the primary MongoDB database that Apostrophe is using or a secondary db? If it's the primary one, are you saying it's configured but not really working properly yet?

It would help if you include the db configuration code from your project. Also make sure to review this reference page: https://v3.docs.apostrophecms.org/reference/modules/db.html#options

**Comment by RyanBreathings** [comment]
What I'm curious about is that I succeeded in connecting Mongo db.
But I want to use the data for the collection I made in Mongo DB, but I don't know what to do.
I want to use data on pages made by apostrophe.

**Comment by boutell** [comment]
@RyanBreathings please see the documentation for async components:

https://v3.docs.apostrophecms.org/guide/async-components.html#defining-async-components

An async component function is an ideal place to await some mongodb calls and return a data object that will be available to the corresponding template.
